schema/site_metadata.json

```diff
@@ -50,6 +58,42 @@
             "type": "number"
         }
       }
+    },
+    "area": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "m2": {
+            "description": "Area of the site in square meters",
+            "type": "number",
+            "examples": [1234, 256, 998765]
+        },
+        "sqf": {
+            "description": "Area of the site in square feet",
+            "type": "number",
+            "examples": [13282.67, 2755.56, 10750617]
+        }
+      }
+    },
+    "site_folder": {
+      "description": "Folder where the UDMI site model is stored",
+      "type": "string",
+      "examples": ["/home/user/udmi_site_model", "https://drive.google.com/corp/drive/folders/iamadrivelocation"]
+    },
+    "source_repo": {
+      "description": "Source repository where the UDMI site model is stored",
+      "type": "string",
+      "examples": ["https://github.com/faucetsdn/udmi_site_model", "git@github.com:faucetsdn/udmi_site_model.git"]
+    },
+    "device_count": {
```

Comment:
i'm wondering if we should generalize this a bit into a bucket, just because I can see the utility it some other counts... e.g. registered devices, discovered-but-not-in-model devices, etc...   I was just thinking maybe put these into a general `count` bucket?

```
count.modeled
count.validated
count.registered
count,discovered
```

(related, `device_count` isn't maybe the best name since they're all _device_ counts of various sorts... it's just the adjective that's describing what _kind_ of device it is -- e.g. modeled devices vs. registered devices, so I'd suggest `modeled` or `total` maybe as the adjective instead of `device`)

Reply:
LGTM, happy to update it if there is agreement.